tests/test_hyperbolic.py

```diff
@@ -446,35 +457,43 @@ def test_scaled_inner_product(self):
         tangent_vec_a = gs.array([1, 2, 3, 4])
         tangent_vec_b = gs.array([5, 6, 7, 8])
         tangent_vec_a = self.space.projection_to_tangent_space(
-            tangent_vec_a, base_point)
+            tangent_vec_a,
+            base_point)
         tangent_vec_b = self.space.projection_to_tangent_space(
-            tangent_vec_b, base_point)
+            tangent_vec_b,
+            base_point)
         scale = 2
         default_space = Hyperbolic(dimension=self.dimension)
         scaled_space = Hyperbolic(dimension=self.dimension, scale=2)
-        inner_product_default_metric = default_space.metric.inner_product(
-            tangent_vec_a, tangent_vec_b, base_point)
-        inner_product_scaled_metric = scaled_space.metric.inner_product(
-            tangent_vec_a, tangent_vec_b, base_point)
+        inner_product_default_metric = \
+            default_space.metric.inner_product(
+                tangent_vec_a,
+                tangent_vec_b,
+                base_point)
+        inner_product_scaled_metric = \
+            scaled_space.metric.inner_product(
+                tangent_vec_a,
+                tangent_vec_b,
+                base_point)
         result = inner_product_scaled_metric
         expected = scale ** 2 * inner_product_default_metric
         self.assertAllClose(result, expected)
 
     @geomstats.tests.np_only
     def test_scaled_squared_norm(self):
         base_point_intrinsic = gs.array([1, 1, 1])
-        base_point = self.space.intrinsic_to_extrinsic_coords(
-            base_point_intrinsic)
+        base_point = \
```

Comment:
Does this also stem from the docstring PR? I'd rather prefer the old formatting with the function call on the first whole line, arguments on the second line.

Reply:
Which is also what our contributing file says